src/app/admin/components/sidebar/sidebar.component.html

```diff
@@ -5,12 +5,30 @@
     </a>
   </div>
   <div class="sidebar__bottom">
-    <div class="sidebar__menu">
-      <ul>
-        <!-- <a routerLink="analytics" routerLinkActive="sidebar__menu--active"><li><i class="far fa-chart-line"></i>Analytics</li></a> -->
-        <a routerLink="learning-objects" routerLinkActive="sidebar__menu--active"><li><i class="far fa-archive"></i>Objects</li></a>
-        <!-- <a routerLink="users" routerLinkActive="sidebar__menu--active"><li><i class="far fa-users"></i>Users</li></a> -->
-      </ul>
+    <div *ngIf="initialized" class="sidebar__menu">
+      <ng-container *ngTemplateOutlet="!editorMode ? collectionMenu : sidebarMenu"></ng-container>
     </div>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<ng-template #sidebarMenu>
```

Comment:
This template is the sidebar menu exclusively for admins, right?

Reply:
No, this template is the markup for menu options since it gets reused in two different places.

Reply:
Ah, I got it now. Thanks!